### PR TITLE
Jeffn/support us header

### PR DESCRIPTION
**Pull request by jnakaso**
## What

Support for "Support us" menu item in common header

1. Support menu item for mobile & desktop
2. Added Feature flag "support-us"
3. Fix path for all pages

## Why Do

> So users can donate.

## How Do

> Feature flag is switchable in Sanity

## Show Me

Screenshot 2024-07-18 at 10 58 24 AM

### CODE REVIEW

File: components/CommonHeader.tsx [hunk 1]
```diff
@@ -106,62 +189,26 @@ const CommonHeader = () => {
               padding: '1.25rem 2.5rem !important',
             }}
           >
-            <LogoBox/>
+            <LogoBox />
             <nav>
               <ul>
```
Comment (mistyb01) [inline components/CommonHeader.tsx:194]: I think there should be a little gap between the links. 

I played a bit with dev tools. You could replace the ul with something like `<List sx={{display: 'flex', gap: '0.5rem'}}>`

image

**Review by mistyb01 — APPROVED** [review]
Looks great! Added a few nitpicks.

File: components/CommonHeader.tsx [hunk 2]
```diff
@@ -13,41 +13,122 @@ import Link from '@mui/material/Link'
...
+        : 'primary'
+      : isCurrent(menuItem)
+        ? 'secondary'
+        : 'secondary';
+
+    const linkColor = ('primary' === menuItem.style)
+      ? theme.palette.primary.contrastText
+      : theme.palette.secondary.contrastText;
+
+    return (<Button
```
[the remark this answers is not in the document]
Comment (jnakaso) [reply components/CommonHeader.tsx]: thx

File: components/CommonHeader.tsx [hunk 3]
```diff
@@ -85,14 +166,16 @@ const CommonHeader = () => {
...
           >
             {/* Hamburger menu when the screen is small. */}
-            <LogoBox/>
+            <LogoBox />
             <IconButton
               size="large"
-              aria-hidden="true"
+              aria-label="page-info"
+              aria-controls="menu-appbar"
+              aria-haspopup="true"
```
[the remark this answers is not in the document]
Comment (jnakaso) [reply components/CommonHeader.tsx]: done

File: components/CommonHeader.tsx (hunk 1 above)
Comment (jnakaso) [reply components/CommonHeader.tsx]: thanks again

**Review by mistyb01 — APPROVED** [review]
Thanks :)

**Review by ybalenko — COMMENTED** [review]
@jnakaso are we working with feature/support-us branch? The reason I am asking is that in your jeffn/support-us-header branch support-us page has a typo (does not have -us).

File: components/CommonHeader.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (ybalenko) [reply components/CommonHeader.tsx]: 'Support **U**s' should be aligned with other buttons like 'Partner With Us'.

example

**Comment by jnakaso** [comment]
> @jnakaso are we working with feature/support-us branch? The reason I am asking is that in your jeffn/support-us-header branch support-us page has a typo (does not have -us).

Reverting to support_us & and also sanitized the existing page.

**Pull request merged**